irohad/torii/impl/command_service.cpp

```diff
@@ -98,9 +98,16 @@ namespace torii {
     tx_processor_->transactionHandle(iroha_tx);
   }
 
-  void CommandService::StatusAsync(
-      iroha::protocol::TxStatusRequest const &request,
-      iroha::protocol::ToriiResponse &response) {
+  grpc::Status CommandService::Torii(
+      grpc::ServerContext *context,
+      const iroha::protocol::Transaction *request,
+      google::protobuf::Empty *response) {
+    Torii(*request, *response);
+    return grpc::Status::OK;
```

Comment:
Why do we always return OK status ?

Reply:
Well, we should return some status just because we overload method from grpc internals and its signature requires that. I do not see here any cases to return a status different from OK.